docker/build_dagmc.sh

```diff
@@ -13,7 +13,6 @@ fi
 rm -rf ${dagmc_build_dir}/bld ${dagmc_install_dir}
 mkdir -p ${dagmc_build_dir}/bld
 cd ${dagmc_build_dir}
-#git clone https://github.com/svalinn/DAGMC -b develop
```

Comment:
The reason these lines were commented is that they would normally be part of a DAGMC build script, but in this specific case, we get the DAGMC source code by copying it into the docker image, not cloning it from github.

Reply:
@ljacobson64 is there any reason to leave this line in as a comment w/ justification or can we just remove it all together? (see [here](https://github.com/svalinn/DAGMC/pull/660#discussion_r368664684))

Reply:
If I understand correctly, there are there just because the normal install `DAGMC` procedure should include them. But in this case as we copy the `DAGMC` from the PR (using `Travis`) we don't need them.

In this case I don't think they should be there.
Those scripts are dedicated for CI not to show people how to install `DAGMC`.
It is probably a good idea to have a version of them somewhere to facilitate `DAGMC` install but for me this is definitely not the place.

Reply:
I agree w/ this assessment @bam241. I think the lines here should be removed and if we want a docker to help users, we need to host that elsewhere.

Reply:
For maintainability purposes, I think it would be good to have some indication of why the source code already exists in this directory. (The current situation with just a commented `git clone` line is also not very helpful.) Perhaps a comment line that says

```
# DAGMC source code already exists in this directory because it was copied into
# the docker image by travis.yml
```

Reply:
I think I understand,

the line is left there to highlight that we don't need to copy DAGMC source, while we normally would in a standard installation process.
So the line is basically there for educational purposes...
Is that correct @ljacobson64 ?

I would definitely prefer to not mix purposes in those scripts. So if there are used for CI install and testing, I would remove the git clone line and add the comment at the head of the file.